submodules/storage/outputs.tf

```diff
@@ -16,12 +16,17 @@ output "s3_buckets" {
   } }
 }
 
-output "s3_policy" {
-  description = "s3 IAM Policy arn"
-  value       = aws_iam_policy.s3.arn
+output "iam_policies" {
+  description = "IAM Policy ARNs"
+  value       = [aws_iam_policy.s3.arn, aws_iam_policy.ecr.arn]
 }
 
 output "efs_security_group" {
   description = "EFS security group id"
   value       = aws_security_group.efs.id
 }
+
+output "container_registry" {
+  description = "ECR base registry URL"
+  value       = trimsuffix(aws_ecr_repository.environment.repository_url, "/environment")
```

Comment:
Don't know if I love this, but a split, splice, join didn't seem much better

Reply:
no, I had to track down what you were doing. I get that we are creating 2 private repos with fixed namespaces that map onto what is put together in nucleus. Fragile but it works.